multinode_integration_tests/src/masq_real_node.rs

```diff
@@ -1415,6 +1435,7 @@ mod tests {
             .build();
 
         assert_eq!(result.neighborhood_mode, neighborhood_mode);
+        assert_eq!(result.min_hops_count, Hops::TwoHops);
```

Comment:
With the `use` pattern at the time of this writing, you shouldn't need `Hops::TwoHops` here; just `TwoHops` should be enough.